components/admin_console/billing/plan_details/feature_list.tsx

```diff
@@ -115,10 +115,6 @@ const FeatureList = (props: FeatureListProps) => {
             id: 'admin.billing.subscription.planDetails.features.mfa',
             defaultMessage: 'Multi-Factor Authentication (MFA)',
         }),
-        intl.formatMessage({
-            id: 'admin.billing.subscription.planDetails.features.multilanguage',
-            defaultMessage: 'Multi-language translations',
-        }),
```

Comment:
Is this removal intended? This still applies to all Starter plans and the System Console card wouldn't require any changes